cypress/integration/heading.feature

```diff
@@ -0,0 +1,105 @@
+Feature: Heading component
+  I want to change Heading component properties
+
+  Background: Open Heading component component page
```

Comment:
one component word is unnecessary, Heading component it's called Heading

Reply:
fixed